libs/features/creature/src/creature-text/creature-text.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { MultiRowEditorComponent } from '@keira/shared/base-abstract-classes';
+import { CreatureText } from '@keira/shared/acore-world-model';
+import { CreatureHandlerService } from '../creature-handler.service';
+import { CreatureTextService } from './creature-text.service';
+import { NgxDatatableModule } from '@swimlane/ngx-datatable';
+import { EditorButtonsComponent } from '@keira/shared/base-editor-components';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { QueryOutputComponent } from '@keira/shared/base-editor-components';
+import { TranslateModule } from '@ngx-translate/core';
+import { TopBarComponent } from '@keira/shared/base-editor-components';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'keira-creature-text',
+  templateUrl: './creature-text.component.html',
+  styleUrls: ['./creature-text.component.scss'],
```

Comment:
since this is empty, the file `creature-text.component.scss` can be deleted and this line removed